locales/base/translation.json

```diff
@@ -2845,5 +2845,16 @@
     "ctaAction_Swap": "Swap smaller amount",
     "ctaAction_Deposit": "Deposit smaller amount",
     "ctaAction_Withdraw": "Withdraw smaller amount"
-  }
+  },
+  "reviewTransaction": {
+    "totalAmount_mutlipleTokens_noFiatPrice": "≈ {{amount1}} {{symbol1}} + {{amount2}} {{symbol2}}"
+  },
+  "tokenAmount": "{{tokenAmount}} {{tokenSymbol}}",
+  "tokenAmountApprox": "≈ {{tokenAmount}} {{tokenSymbol}}",
+  "localAmount": "{{localCurrencySymbol}}{{localAmount}}",
+  "localAmountApprox": "≈ {{localCurrencySymbol}}{{localAmount}}",
+  "tokenAndLocalAmount_oneToken": "{{tokenAmount}} {{tokenSymbol}} <0>({{localCurrencySymbol}}{{localAmount}})</0>",
+  "tokenAndLocalAmount_oneToken_noFiatPrice": "{{tokenAmount}} {{tokenSymbol}}",
```

Comment:
nit: could we reuse the "tokenAmount" string? same with `tokenAndLocalAmountApprox_oneToken_noFiatPrice`, can that use "tokenAmountApprox" to reduce the amount of strings that do similar things?